e2e/unsubscribed.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { test, expect, type Page } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+import metaTags from '../client/i18n/locales/english/meta-tags.json';
+
+let page: Page;
+
+test.afterAll(async () => {
+  await page.close();
+});
+
+test.describe('The unsubscribed page without unsubscribeId', () => {
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage();
+    await page.goto('/unsubscribed');
+  });
```

Comment:
```suggestion
test.beforeEach(async ({ page }) => {
  await page.goto('/unsubscribed');
});

test.afterEach(async ({ page }) => {
  await page.close();
});

test.describe('The unsubscribed page without unsubscribeId', () => {
```

We should ideally run tests in isolation.